src/Engine/Manager/ScriptManager.cpp

```diff
@@ -70,16 +118,9 @@ void AngelScriptDebugLineCallback(asIScriptContext *ctx, const std::map<std::str
     if (breakpoints->find(fileName) != breakpoints->end() && breakpoints->at(fileName).find(line) != breakpoints->at(fileName).end()) {
         // A break point has been reached so the execution of the script should be suspended
         // Show the call stack
-        for (asUINT n = 0; n < ctx->GetCallstackSize(); n++) {
-            asIScriptFunction *func;
-            const char *scriptSection;
-            int line, column;
-            func = ctx->GetFunction(n);
-            line = ctx->GetLineNumber(n, &column, &scriptSection);
-            printf("%s:%s:%d,%d\n", scriptSection,
-                func->GetDeclaration(),
-                line, column);
-        }
+        std::string callstack = CallstackToString(ctx);
+        std::string variables = VariablesToString(ctx, 0);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Variable 'variables' is assigned a value that is never used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunreadVariable)